views/user_registrations/new.haml

```diff
@@ -0,0 +1,11 @@
+.login-form
+  %form.form-signin{action: '/user_registrations/create', method: 'POST'}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace